### PR TITLE
docs: add section explaining how to convert enum from/to str/int values

### PR DESCRIPTION
**Pull request by machel2**
docs.md - enums - add section about converting enum from/to string and int values

Covers usage of `from()` `froms_string()` and `str()` functions

### CODE REVIEW

**Comment by JalonSolov** [comment]
Question:  Now that `from` is implemented and working, should `from_string` be deprecated?

**Comment by spytheman** [comment]
> Question: Now that `from` is implemented and working, should `from_string` be deprecated?

yes

File: doc/docs.md [hunk 1]
```diff
@@ -3405,6 +3405,33 @@ three
...
+enum Cycle {
+	one
+	two = 2
+	three
+}
+
+// Create enum from value
+println(Cycle.from(2) or { Cycle.three })
+println(Cycle.from('two')!)
+println(Cycle.from_string('two') or { panic('unknown value') })
```
Comment (spytheman) [inline doc/docs.md:3420]:
```suggestion
```

File: doc/docs.md [hunk 2]
```diff
@@ -3405,6 +3405,32 @@ three
...
+	one
+	two = 2
+	three
+}
+
+// Create enum from value
+println(Cycle.from(2) or { Cycle.three })
+println(Cycle.from('two')!)
+
+// Enum to tring
```
Comment (spytheman) [inline doc/docs.md:3421]:
```suggestion
// Enum to string
```

File: doc/docs.md [hunk 3]
```diff
@@ -3405,6 +3405,32 @@ three
...
+	one
+	two = 2
+	three
+}
+
+// Create enum from value
+println(Cycle.from(2) or { Cycle.three })
+println(Cycle.from('two')!)
+
+// Converting an enum to a string
```
Comment (spytheman) [inline doc/docs.md:3421]:
```suggestion
// Convert an enum value to a string
```

File: doc/docs.md [hunk 4]
```diff
@@ -3405,6 +3405,32 @@ three
...
+
+```v
+enum Cycle {
+	one
+	two = 2
+	three
+}
+
+// Create enum from value
+println(Cycle.from(2) or { Cycle.three })
```
Comment (spytheman) [inline doc/docs.md:3418]:
```suggestion
println(Cycle.from(1) or { Cycle.three })
```

File: doc/docs.md [hunk 5]
```diff
@@ -3405,6 +3405,32 @@ three
...
+println(Cycle.from('two')!)
+
+// Convert an enum value to a string
+println(Cycle.two.str())
+```
+
+Output:
+
+```
+two
```
Comment (spytheman) [inline doc/docs.md:3428]:
```suggestion
three
```

File: doc/docs.md [hunk 6]
```diff
@@ -3405,6 +3405,32 @@ three
...
+// Convert an enum value to a string
+println(Cycle.two.str())
+```
+
+Output:
+
+```
+three
+two
+two
```
Comment (spytheman) [inline doc/docs.md:3430]:
```suggestion
```

File: doc/docs.md [hunk 7]
```diff
@@ -3405,6 +3405,31 @@ three
...
+	two = 2
+	three
+}
+
+// Create enum from value
+println(Cycle.from(10) or { Cycle.three })
+println(Cycle.from('two')!)
+
+// Convert an enum value to a string
+println(Cycle.two.str())
```
Comment (spytheman) [inline doc/docs.md:3422]:
```suggestion
println(Cycle.one.str())
```

File: doc/docs.md [hunk 8]
```diff
@@ -3405,6 +3405,31 @@ three
...
+// Convert an enum value to a string
+println(Cycle.one.str())
+```
+
+Output:
+
+```
+three
+two
+two
```
Comment (spytheman) [inline doc/docs.md:3430]:
```suggestion
one
```

**Pull request merged**